Tests/AppTests/Mocks/HomeIndexModel+mock.swift

```diff
@@ -4,6 +4,7 @@
 extension HomeIndex.Model {
     static var mock: HomeIndex.Model {
         .init(
+            stats: .init(packageCount: 2544, versionCount: 38410),
```

Comment:
This line here ensures we recreate the exact same snapshot as before. It's a good test to change the numbers here and see the snapshot test fail.